samples/wire/README.md

```diff
@@ -0,0 +1,317 @@
+# Wire Tutorial
+
+Let's learn to use Wire by example. The [Wire README][readme] provides thorough
+documentation of the tool's usage. For readers eager to jump right in, the
```

Comment:
s/jump right in/see Wire applied to a cloud server/

By being more precise, we clarify who the audience for this sample is by contrast: this is just Wire by itself.

Reply:
Well said. Done.